_data/errors/extraction/databases/mysql.yml

```diff
@@ -244,4 +246,19 @@ all:
 
        The version of the MySQL database is less than 5.6.2, which is the minimum version that supports binlog replication. Versions prior to 5.6.2 don't support the binlog replication feature, which is required to use Log-based Incremental Replication in Stitch.
     fix-it: |
-      The source MySQL database must be running MySQL version 5.6.2 or greater to use binlog replication. If you can upgrade your database version, you may be able to use binlog replication.
\ No newline at end of file
+      The source MySQL database must be running MySQL version 5.6.2 or greater to use binlog replication. If you can upgrade your database version, you may be able to use binlog replication.
+
+## Empty table error
+  - message: *empty-table-binlog
+    level: "critical"
+    category: "Log-based Incremental Replication"
+    category-doc: |
+      {{ link.replication.log-based-incremental | prepend: site.baseurl | append: "#log-based-incremental-replication-terminology" }}
+    version: "1"
+    id: "empty-table-binlog"
+    applicable-to: *all-mysql
+    summary: "There are no primary keys in the table."
+    cause: |
+      The initial historical replication of the table is empty. For Log-based Incremental Replication there must be primary keys in the table. If the table is empty, there are no primary keys, therefore this error will occur. Remaining tables will not be replicated, even if they contain valid data, if this error occurs.
```

Comment:
1. It's not the replication that's empty - it's the table in the data source that's empty.
2. Why are Primary Keys required?
3. I recommend making sure that the bit about other tables not being replicated really stands out - how can you do that?
4. Please capitalize `Primary Keys`